### PR TITLE
planner: update SPM doc 

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16770
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-plan-management.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ To make the execution plan of a SQL statement fixed to a historical execution pl
 When using this feature, note the following:
 
 - The feature generates hints according to historical execution plans and uses the generated hints for binding. Because historical execution plans are stored in [Statement Summary Tables](/statement-summary-tables.md), before using this feature, you need to enable the [`tidb_enable_stmt_summary`](/system-variables.md#tidb_enable_stmt_summary-new-in-v304) system variable first.
-- This feature does not support TiFlash queries, Join queries with three or more tables, and queries that contain subqueries.
+- For TiFlash queries, Join queries with three or more tables, and queries that contain subqueries, auto-generated hints might not be complete and able to fix the plan completely, for such cases a warning will be outputted.
```
Comment (hfxsd) [inline sql-plan-management.md:239]:
```suggestion
- For TiFlash queries, Join queries with three or more tables, and queries that contain subqueries, the auto-generated hints are not complete enough, which might result in the plan not being fully bound. In such cases, a warning will occur when creating a binding.
```

File: sql-plan-management.md [hunk 2]
```diff
@@ -484,6 +484,52 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## Utilize Statement Summary table to obtain queries that need to be bound
```
Comment (hfxsd) [inline sql-plan-management.md:487]:
```suggestion
## Utilize the Statement Summary table to obtain queries that need to be bound
```

File: sql-plan-management.md [hunk 3]
```diff
@@ -484,6 +484,52 @@ SHOW binding_cache status;
...
+
+[Statement Summary](/statement-summary-tables.md) stores recent SQL execution information, such as latency, execution times, corresponding query plans, etc. You can query Statement Summary to get qualified `plan_digest`, and [create bindings according to these historical execution plans](/sql-plan-management.md#create-a-binding-according-to-a-historical-execution-plan).
```
Comment (hfxsd) [inline sql-plan-management.md:489]:
```suggestion
[Statement Summary](/statement-summary-tables.md) stores recent SQL execution information, such as latency, execution times, and corresponding query plans. You can query Statement Summary to get qualified `plan_digest`, and then [create bindings according to these historical execution plans](/sql-plan-management.md#create-a-binding-according-to-a-historical-execution-plan).
```

File: sql-plan-management.md [hunk 4]
```diff
@@ -484,6 +484,52 @@ SHOW binding_cache status;
...
+  AND stmt_type = 'Select'                                     -- Only consider select statements
+  AND schema_name NOT IN ('INFORMATION_SCHEMA', 'mysql')       -- Not an internal query
+  AND plan_in_binding = 0                                      -- No binding yet
+GROUP BY stmts.`digest`
+  HAVING COUNT(DISTINCT(stmts.plan_digest)) > 1                -- This query is unstable. It has more than 1 plan.
+         AND SUM(exec_count) > 10                              -- High-frequency, and has been executed more than 10 times.
+ORDER BY SUM(exec_count) DESC LIMIT 100;                       -- Top 100 high-frequency queries.
+```
+
+By applying certain filtering conditions to obtain queries that meet the criteria, you can then directly execute the statements corresponding to the `binding_stmt` column to create bindings.
```
Comment (hfxsd) [inline sql-plan-management.md:522]:
```suggestion
By applying certain filtering conditions to obtain queries that meet the criteria, you can then directly execute the statements in the corresponding `binding_stmt` column to create bindings.
```

File: sql-plan-management.md [hunk 5]
```diff
@@ -236,7 +236,7 @@ To make the execution plan of a SQL statement fixed to a historical execution pl
 When using this feature, note the following:
 
 - The feature generates hints according to historical execution plans and uses the generated hints for binding. Because historical execution plans are stored in [Statement Summary Tables](/statement-summary-tables.md), before using this feature, you need to enable the [`tidb_enable_stmt_summary`](/system-variables.md#tidb_enable_stmt_summary-new-in-v304) system variable first.
-- This feature does not support TiFlash queries, Join queries with three or more tables, and queries that contain subqueries.
+- For TiFlash queries, Join queries with three or more tables, and queries that contain subqueries, the auto-generated hints are not complete enough, which might result in the plan not being fully bound. In such cases, a warning will occur when creating a binding.
```
Comment (songrijie) [inline sql-plan-management.md:239]:
```suggestion
- For TiFlash queries, Join queries with three or more tables, and queries that contain subqueries, the auto-generated hints are not adequate, which might result in the plan not being fully bound. In such cases, a warning will occur when creating a binding.
```

File: sql-plan-management.md [hunk 6]
```diff
@@ -484,6 +484,52 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## Utilize the Statement Summary table to obtain queries that need to be bound
+
+[Statement Summary](/statement-summary-tables.md) stores recent SQL execution information, such as latency, execution times, and corresponding query plans. You can query Statement Summary to get qualified `plan_digest`, and then [create bindings according to these historical execution plans](/sql-plan-management.md#create-a-binding-according-to-a-historical-execution-plan).
+
+The following example searches for SELECT statements that have been executed more than 10 times in the past two weeks, have unstable execution plans, and have not been bound. It sorts the queries by the execution times, and binds the top 100 queries to their fastest plans.
```
Comment (songrijie) [inline sql-plan-management.md:491]:
```suggestion
The following example searches for SELECT statements that have been executed more than 10 times in the past two weeks, have multiple execution plans without SQL binding. It sorts the queries by the execution times, and binds the top 100 queries to their fastest plans.
```

**Review by songrijie — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: sql-plan-management.md [hunk 7]
```diff
@@ -484,6 +484,52 @@ SHOW binding_cache status;
 1 row in set (0.00 sec)
 ```
 
+## Utilize the Statement Summary table to obtain queries that need to be bound
+
+[Statement Summary](/statement-summary-tables.md) stores recent SQL execution information, such as latency, execution times, and corresponding query plans. You can query Statement Summary to get qualified `plan_digest`, and then [create bindings according to these historical execution plans](/sql-plan-management.md#create-a-binding-according-to-a-historical-execution-plan).
+
+The following example searches for SELECT statements that have been executed more than 10 times in the past two weeks, and have multiple execution plans without SQL binding. It sorts the queries by the execution times, and binds the top 100 queries to their fastest plans.
```
Comment (Oreoxmt) [inline sql-plan-management.md:491]:
```suggestion
## Utilize the statement summary table to obtain queries that need to be bound

[Statement summary](/statement-summary-tables.md) records recent SQL execution information, such as latency, execution times, and corresponding query plans. You can query statement summary tables to get qualified `plan_digest`, and then [create bindings according to these historical execution plans](/sql-plan-management.md#create-a-binding-according-to-a-historical-execution-plan).

The following example queries `SELECT` statements that have been executed more than 10 times in the past two weeks, and have multiple execution plans without SQL binding. It sorts the queries by the execution times, and binds the top 100 queries to their fastest plans.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 8]
```diff
@@ -3356,6 +3356,17 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+### `tidb_load_binding_timeout` <span class="version-mark">New in v8.0.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `200`
+- Range: `(0, 2147483647]`
+- Unit: Milliseconds
+- This variable is used to control the timeout of binding loading. If the time of binding loading exceeds this variable, it will stop.
```
Comment (hfxsd) [inline system-variables.md:3368]:
```suggestion
- This variable is used to control the timeout of binding loading. If the execution time of loading bindings exceeds this value, the loading will be stopped.
```

File: system-variables.md [hunk 9]
```diff
@@ -3356,6 +3356,17 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+### `tidb_load_binding_timeout` <span class="version-mark">New in v8.0.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `200`
+- Range: `(0, 2147483647]`
+- Unit: Milliseconds
+- This variable is used to control the timeout of binding loading. If the execution time of loading bindings exceeds this value, the loading will be stopped.
```
Comment (hfxsd) [inline system-variables.md:3368]:
```suggestion
- This variable is used to control the timeout of loading bindings. If the execution time of loading bindings exceeds this value, the loading will stop.
```

**Pull request merged**